MaxText/pyconfig.py

```diff
@@ -129,6 +139,33 @@ def user_init(raw_keys):
 
     validate_attention_type(raw_keys['attention'])
 
+  @staticmethod
+  def update_model_vars(raw_keys):
+    ''' Update model config variables
+    '''
+    validate_model_name(raw_keys['model_name'])
```

Comment:
I don't think we quite are asserting that the models are the same yet. I think for now a polite solution would be to add a ValueError for if you used llama-7b.